indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -118,7 +118,15 @@ def _process(self):
                 _('Download Attachments'),
                 'attachment',
                 type='href-custom',
-                url=url_for('.registrations_attachments_export', regform),
+                url=url_for('.registrations_attachments_export', regform, flatten=False),
```

Comment:
`False` is the default so we can omit it the parameter here :)
```suggestion
                url=url_for('.registrations_attachments_export', regform),
```

Reply:
Fixed 👍